### DISCUSSION TITLE
Github Campus expert

### DISCUSSION THREAD

**Discussion by Blackie360** [community/community#110806] [GitHub Education] [answered]
### Hi everyone,

Who is eligible to apply for the campus expert program? What are the criteria used to choose the right applicants? what and how long is the application process?

**Comment by SebastienXia** [comment, marked as answer]
Hi,

I think there will be all the information you need for what you ask. By consulting the GitHub documentation below

[]()https://docs.github.com/fr/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-github-campus-experts

**Comment by Bambobee** [comment]
Yeah

**Comment by Blackie360** [comment]
Information in the docs

**Reply by SebastienXia** [reply]
Yes ? Is there a problem?
